stan/math/prim/scal/prob/cauchy_rng.hpp

```diff
@@ -27,7 +27,7 @@ namespace stan {
      * @param rng random number generator
      * @return Cauchy random variate
      * @throw std::domain_error if mu is infinite or sigma is nonpositive
-     * @throw std::invalid_argument if non-scalars arguments are of different
+     * @throw std::invalid_argument if non-scalar arguments are of different
```

Comment:
I strongly prefer "size" to "length" here (I know this isn't yours) as "length" is ambiguous with lengths of vectors.

Reply:
Makes sense. I went through and changed this in the rngs I've modified so far.